### PR TITLE
Function reformatting for set_queue_timeout

### PR DESCRIPTION
**Pull request by AbdulrahmanHammouda**
set_queue_timeout #2549

### CODE REVIEW

File: ivy/functional/ivy/general.py [hunk 1]
```diff
@@ -2065,7 +2065,9 @@ def get_all_arrays_in_memory():
 
 
 def num_arrays_in_memory():
-    """Returns the number of arrays which are currently alive."""
+    """
```
Comment (rush2406) [inline ivy/functional/ivy/general.py:2068]: This function need not be reformatted, can you please revert the change?

File: ivy/functional/ivy/general.py [hunk 2]
```diff
@@ -2076,14 +2078,29 @@ def print_all_arrays_in_memory():
...
-    function being called is 10 seconds.
+    """
+    Set the global queue timeout value (in seconds) 
+    Default value without this function being called is 15 seconds.
 
     Parameters
     ----------
     timeout
-        The timeout to set in seconds.
+        The timeout when waiting for containers to arrive from the queues. To be set in seconds.
```
Comment (rush2406) [inline ivy/functional/ivy/general.py:2088]: The ` lint test` is failing because the line is > 88 chars. Can you please reformat it so that the tests pass?

**Review by rush2406 — CHANGES_REQUESTED** [review]
Hi @AbdulrahmanHammouda 
Thanks for contributing! There is some minor reformatting to be done, can you please address the comments and request a review when done

**Comment by AbdulrahmanHammouda** [comment]
Reformatting for set_queue_timeout #2549 from Reformat general #791

File: ivy/functional/ivy/general.py (hunk 2 above)
Comment (AbdulrahmanHammouda) [reply ivy/functional/ivy/general.py]: Yes all done. Let me know if there is anything I can do

**Comment by rush2406** [comment]
Thanks @AbdulrahmanHammouda , I have merged the PR
